### PR TITLE
Fix image names

### PR DESCRIPTION
**Pull request by chentex**
### Description

Inconsistencies on naming have been having negative effects on our automation on image building.

- Rename iperf3 image to iperf
- Rename ycsb-server image to ycsb
- Rename fs-drift image to fs_drift
- Rename fs-drift tool to fs_drift

### Fixes

https://github.com/cloud-bulldozer/benchmark-wrapper/issues/347

### CODE REVIEW

File: tests/test_list [hunk 1]
```diff
@@ -1,7 +1,7 @@
 test_backpack.sh
 test_ycsb.sh
 test_fiod.sh
-test_fs-drift.sh
+test_fs_drift.sh
 test_smallfile.sh
 test_pgbench.sh
 test_iperf3.sh
```
Comment (learnitall) [inline tests/test_list:7]: Do we need to change iperf3 -> iperf here as well?

**Review by learnitall — CHANGES_REQUESTED** [review]
Took a look, here are the main things I notice:

* Role names weren't changed. Do we want to change these too? Might as well right since we're at it?
* scale tool name wasn't changed in the scale_openshift template (https://github.com/cloud-bulldozer/benchmark-operator/blob/ae117009213f322b069a324649ed9916b41a006d/roles/scale_openshift/templates/scale.yml#L83)
* iperf test name is still at iperf3

Tomorrow I'll try to do a more thorough look with grep to see if there are other things we're missing here or want to take into consideration.

**Comment by jtaleric** [comment]
Still relevant... Let's work to close this out in the new year

**Comment by jtaleric** [comment]
still relevant

**Comment by rsevilla87** [comment]
Hey @chentex, seems we're hitting this error in CI
```
   hammerdb-standard35/39 in 1586 sec ✓ hammerdb-standard [242122]
   fs_drift-standard36/39 in 1586 sec ✗ fs_drift-standard [1675]
   (in test file ./017-fs_drift.bats, line 13)
     `envsubst < ${CR} | kubectl apply -f -' failed
   The Benchmark "fs_drift-benchmark" is invalid: metadata.name: Invalid value: "fs_drift-benchmark": a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')
   fs_drift-hostpath37/39 in 1586 sec ✗ fs_drift-hostpath [1799]
   (in test file ./017-fs_drift.bats, line 23)
     `envsubst < ${CR} | kubectl apply -f -' failed
   The Benchmark "fs_drift-hostpath-benchmark" is invalid: metadata.name: Invalid value: "fs_drift-hostpath-benchmark": a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')
```

**Comment by chentex** [comment]
Hey, Raúl, thanks for letting me know, I think this is still a WIP, there is a team meeting that we need to have to discuss the names and the way we were automatically building the images, not everyone was onboard on changing them, I will add them to this weeks meeting to take on it again.

**Pull request closed without merging**